src/Routes.js

```diff
@@ -2,13 +2,15 @@ import React from "react";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 
 import Main from "./pages/Main/Main";
+import Footer from "./components/Footer/Footer";
 
 class Routes extends React.Component {
     render() {
         return (
             <Router>
                 <Switch>
                     <Route exact path="/" component={Main} />
+                    <Route exact path="/Footer" component={Footer} />
                 </Switch>
             </Router>
```

Comment:
```suggestion
                </Switch>
              <Footer/>
            </Router>
```
Footer 컴포넌트가 어느 페이지에서든 보이는 컴포넌트라면 Switch와 Router사이에 위치시켜주세요!